src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
+      <label>
+        <input
+          type="text"
+          class="search-bar search-bar--big"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles”"
+        />
+      </label>
+    </form>
+    <form
+      action="#"
+      data-qa="small"
+    >
```

Comment:
Similar to the first form, the `form` element here is self-closing. Ensure that the form has both an opening and a closing tag by changing `<form ... />` to `<form ...></form>`.